go/BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("//go:rules.bzl", "swig_native_go_library")
+
+swig_native_go_library(
+    name = "typedb_driver_go_native",
+    lib = "//c:typedb_driver_clib_headers",
+    package_name= "typedb_driver",
+    interface = "//c:typedb_driver.i",
+    includes = ["//c:swig/typedb_driver_go.swg"],
+    enable_cxx = True,
+    visibility = ["//visibility:public"],
+)
+
+# SAMPLE CODE FOR TESTING. THIS SHOULD BE USED AS A GUIDE WHEN DEVELOPING PRODUCTION CODE.
```

Comment:
We can add TODO for better search capabilities (and it's also highlighted in yellow in IDEA!)

Reply:
But I guess we'll remove this target as you'll have your driver's base working tomorrow, so it's fine.